code/modules/mod/mod_types.dm

```diff
@@ -82,6 +82,7 @@
 		/obj/item/mod/module/t_ray,
 		/obj/item/mod/module/quick_carry,
 		/obj/item/mod/module/headprotector,
+		/obj/item/mod/module/mister/atmos,
 	)
 	default_pins = list(
 		/obj/item/mod/module/magboot,
```

Comment:
```suggestion
		/obj/item/mod/module/magboot,
		/obj/item/mod/module/mister/atmos,
```

It would be nice to have the mister as a default pin on the top bar.
A lot of people won't even realize the MODsuit has the function built in, it saves times pinning it ourselves or opening/closing the UI to use it.

Reply:
Personally I always enabled it from the wheel but I see no reason why not